lua/lspconfig/tailwindcss.lua

```diff
@@ -0,0 +1,87 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = 'tailwindcss'
+local bin_name = 'tailwindcss-language-server'
+
+configs[server_name] = {
+  default_config = {
+    cmd = {bin_name, '--stdio'},
+    -- filetypes copied and adjusted from tailwindcss-intellisense
+    filetypes = {
+      -- html
+      'aspnetcorerazor',
+      'astro',
+      'astro-markdown',
+      'blade',
+      'django-html',
+      'edge',
+      'eelixir', -- vim ft
+      'ejs',
+      'erb',
+      'eruby', -- vim ft
+      'gohtml',
+      'haml',
+      'handlebars',
+      'hbs',
+      'html',
+      -- 'HTML (Eex)',
+      -- 'HTML (EEx)',
+      'html-eex',
+      'jade',
+      'leaf',
+      'liquid',
+      'markdown',
+      'mdx',
+      'mustache',
+      'njk',
+      'nunjucks',
+      'php',
+      'razor',
+      'slim',
+      'twig',
+      -- css
+      'css',
+      'less',
+      'postcss',
+      'sass',
+      'scss',
+      'stylus',
+      'sugarss',
+      -- js
+      'javascript',
+      'javascriptreact',
+      'reason',
+      'rescript',
+      'typescript',
+      'typescriptreact',
+      -- mixed
+      'vue',
+      'svelte',
+    },
+    init_options = {
+      userLanguages = {
+        eelixir = 'html-eex',
+        eruby = 'erb',
+      },
+    },
+    root_dir = function(fname)
+      return util.root_pattern('tailwind.config.js', 'tailwind.config.ts')(fname) or
+      util.root_pattern('postcss.config.js', 'postcss.config.ts')(fname) or
+      util.find_package_json_ancestor(fname) or
+      util.find_node_modules_ancestor(fname) or
+      util.find_git_ancestor(fname)
+    end,
+  },
+  docs = {
+    package_json = 'https://raw.githubusercontent.com/tailwindlabs/tailwindcss-intellisense/master/packages/tailwindcss-language-server/package.json',
+    description = [[
+https://github.com/tailwindlabs/tailwindcss-intellisense
+
+Tailwind CSS Language Server
```

Comment:
Can you add the npm command here? see cssls, htmls, pyright, etc.

Reply:
It's not yet available on npm, afaik.

Reply:
Arch Linux users can install the server using https://aur.archlinux.org/packages/tailwindcss-language-server/

Reply:
It's pretty easy to install manually, if you need to see how for reference nvim-lspinstall https://github.com/kabouzeid/nvim-lspinstall/blob/878645fffab9b14eec104ec95a64c4165b6138e8/lua/lspinstall/servers/tailwindcss.lua#L4-L12 does basically the same thing I did to test this

Reply:
Comment with intent to ship to npm:
https://github.com/tailwindlabs/tailwindcss-intellisense/issues/301#issuecomment-850542512

Work towards that:
https://github.com/tailwindlabs/tailwindcss-intellisense/commit/bdb398bf80a5557b329e5b1c7f2019c58e49cd00

package.json for standalone server:
https://github.com/tailwindlabs/tailwindcss-intellisense/blob/master/packages/tailwindcss-language-server/package.json

Reply:
That's great news! I'd still like to include the basic instructions to install this when we merge it, as that's usually the first question/issue people have when we add a server without it.

Reply:
Is it preferred to suggest building it themselves or extracting the server from the pre-built VS Code extension?

Reply:
I'd probably just do like nvim-lspinstall for now and we'll await the npm package/update when it happens

Reply:
Installation instructions added 👍